src/promptflow/promptflow/_sdk/_serving/_errors.py

```diff
@@ -16,6 +16,24 @@ class MissingRequiredFlowInput(BadRequest):
     pass
 
 
+class FlowConnectionError(UserErrorException):
+    pass
+
+
+class UnsupportedConnectionProvider(FlowConnectionError):
+    def __init__(self, provider):
+        super().__init__(
+            message_format="Unsupported connection provider {provider}, "
```

Comment:
if provider is Callable, there may be exceptions here?

Reply:
what's the meaning by strange?

Reply:
offline synced